### ISSUE TITLE
Fix form for new database creation

### ISSUE THREAD

**Issue by kashif** [nomadlabs/spacialdb#2]

**Comment by lfcj** [comment]
Hey @kashif, this from is under `views/instaces/_form.html`, what other things should we ask the user to be able to created the db?

**Comment by kashif** [comment]
hmm i guess for now the size of the instance, the region should be the most important

**Comment by lfcj** [comment]
cool, region was already 'choosable' from the beginning, and I thought the size was conditioned by the plan. Correct me if I am wrong.

**Comment by kashif** [comment]
yes sure but we still want to encourage the customer to choose again and perhaps go with a higher plan :wink:

**Comment by lfcj** [comment]
Get it, so something like: 
hostname: .....
region: x
            x
            ....
memory: x
               y
               z

and if he chooses memory y, but had chosen a plan that has memory x, then we should lauch a message that says: "Hey, that is not in your plan, wanna updrage?"
